components/common/src/main/java/com/bloxbean/cardano/yaci/store/common/domain/Utxo.java

```diff
@@ -8,30 +8,34 @@
 import java.math.BigInteger;
 import java.util.List;
 
+/**
+ * This class is used to represent UTXO for controller API
+ */
 @Getter
-@NoArgsConstructor
-@AllArgsConstructor
 @Builder
 @ToString
+@NoArgsConstructor
+@AllArgsConstructor
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
-/**
- * This class is used to represent UTXO for controller API
- */
 public class Utxo {
+
     private String txHash;
     private int outputIndex;
     private String address;
     private List<Amount> amount;
     private String dataHash;
     private String inlineDatum;
     private String referenceScriptHash;
+    private Integer epochNo;
+    private Long blockHeight;
```

Comment:
Should we rename it to "blockNumber", as most of the domain classes use "blockNumber" ?
Similarly, we can use "epoch" instead of "epochNo".

Reply:
Sure